src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Tests/Controllers/PublicationsControllerTests.cs

```diff
@@ -356,6 +365,234 @@ private static async Task<HttpResponseMessage> GetPublication(HttpClient client,
         }
     }
 
+    public class ListDataSetsTests : PublicationsControllerTests
+    {
+        public ListDataSetsTests(TestApplicationFactory testApp) : base(testApp)
+        {
+        }
+
```

Comment:
Realised we're missing any tests that could check that our response is actually correct i.e. the data sets map correctly to the view models. It might be a good idea to do this as a separate test case from the one below so it's more focused?

Looks like a similar thing applies for our publication endpoint tests above as well

Reply:
correct! I'll get them added

Reply:
I think it's more succinct to add them as part of the same test though personally? I'd kind of expect it to be there